ci/azure-cross-compile.yml

```diff
@@ -26,6 +34,18 @@ jobs:
         vmImage: ubuntu-16.04
         target: x86_64-sun-solaris
 
+      FreeBSD_32:
+        vmImage: ubuntu-16.04
```

Comment:
Why cross-compile FreeBSD i386 from Linux?  Better to do it from FreeBSD amd64.  That way we can run the tests, too.  See https://github.com/nix-rust/nix/blob/master/.cirrus.yml .

Reply:
@asomers either way, this cross compilation doesn't seem to catch the error?

Reply:
What error are you expecting to catch?

Reply:
This fix: https://github.com/tokio-rs/mio/commit/aeaebb0a908f65910ef5cca2863e71734d77d0aa